modules/messages/client/components/InboxThread.js

```diff
@@ -41,6 +47,7 @@ export default function InboxThread({ user, thread }) {
 InboxThread.propTypes = {
   user: userType.isRequired,
   thread: PropTypes.shape({
+    read: PropTypes.boolean.isRequired,
```

Comment:
I think it's `bool`, not `boolean`

Reply:
Ahh that's right silly me I have been in TypeScript land too long. Hang tight.

Reply:
Ok now we are good. Thanks!